torch/testing/_internal/common_utils.py

```diff
@@ -1566,6 +1566,21 @@ def wrapper(*args, **kwargs):
 
     return decorator
 
+def skipIfRocm(func=None, *, msg="test doesn't currently work on the ROCm stack"):
```

Comment:
this change is not necessary, this function is already implemented in this file https://github.com/ROCm/pytorch/blob/release/2.5/torch/testing/_internal/common_utils.py#L1752